src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -1,14 +1,21 @@
-import { createStandardAction, createAsyncAction } from 'typesafe-actions';
```

Comment:
I seem to have forgotten a console.log in the `updateSearch` action in this file. Could you please remove it?

Reply:
That's done